style.css

```diff
@@ -0,0 +1,6 @@
+main {
+  display: block;
+  margin: auto;
+  max-width: 100%;
+  height: auto;
+}
```

Comment:
Great Job
Please add some colors and images to make for the form more attractive